contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
1. I have a use case right now where the symlinks break things.  (I'm working on a sandboxed mac app that embeds Julia.)
2. It seems to me that if a user knows about INCLUDEDIR, DATAROOTDIR, etc. then the user would know about JULIALIBNAME (if it's called that).  Or if the user knows to append -debug, then JULIALIBNAME would not be much of a stretch to find and use.